packages/live-queries/tests/acceptance/live-query-test.js

```diff
@@ -4,17 +4,19 @@ import { visit, fillIn, click } from '@ember/test-helpers';
 
 import Fixtures from '@cardstack/test-support/fixtures';
 
-let scenario = new Fixtures('default', factory => {
-  factory.addResource('items').withAttributes({content: 'hello'});
+let scenario = new Fixtures({
+  create(factory) {
+    factory.addResource('items').withAttributes({content: 'hello'});
+  },
+
+  destroy() {
+    return [{ type: 'items' }];
```

Comment:
I think this is not needed if we are automatically destroying everything that was made during `create`. Is that implemented?

Reply:
good point. the destroy does automatically remove anything that was in the `create`